internal/checker/cloudformation/rules/logging/verify_cloudtrail_encrypted.rego

```diff
@@ -0,0 +1,48 @@
+# METADATA
+# title: "Verify Cloudtrail logs are encrypted at rest"
+# description: "AWS CloudTrail documents AWS API calls, accessible per IAM policies. AWS KMS manages encryption keys with HSMs. CloudTrail logs use SSE and KMS CMKs for security. Configure CloudTrail with SSE-KMS for added confidentiality. Users need S3 read permission and CMK decryption rights for log access."
+# scope: package
+# related_resources:
+# - https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-resource-cloudtrail-trail.html
+# custom:
+#   id: CB_CFT_29
+#   severity: MEDIUM
+package lib.cloudformation.CB_CFT_29
+import future.keywords.in
+
+resource := "AWS::CloudTrail::Trail"
+
+is_valid {
+	some resources in input.Resources
+	resources.Type == resource
+}
+
+pass[protocol] {
```

Comment:
return properties here instead of protocol.. always if the return is  an object{}